### PR TITLE
Logrus

### PR DESCRIPTION
**Pull request by OlehEngineer**
I have added logging using Logrus package. Logging to the LOG.log file and standard output.
Logging contain next infromation:

1. log level
2. update ID
3. chat ID
4. massage text
5. errors

### CODE REVIEW

File: Telegram_Bot/telegramBot.exe
Comment (Unlym) [inline Telegram_Bot/telegramBot.exe:1]: please add that to  .gitignore

**Comment by Unlym** [comment]
@OlehEngineer, please separate the main in a few files, maybe also pkg, I see that like:
1.  config pkg with config.go file
2. bot pkg where we have connected to the bot and methods for it.
   - example

    >    type Bot struct {
    > 	bot *<lib>.BotAPI
    > }
    > 
    > func NewBot(bot *<lib>.BotAPI) *Bot {
    > 	return &Bot{bot: bot}
    > }
    > 
    > func (b *Bot) Start() error 

3. and also when you start implementing HTTP client to get info about holidays, also do it in another pkg
4. **main.go** must be a simple file with a minimum lines of code

File: Telegram_Bot/telegramBot.exe
Comment (OlehEngineer) [reply Telegram_Bot/telegramBot.exe:1]: yes, it make sense.

**Comment by OlehEngineer** [comment]
> 1. config pkg with config.go file

There is **.env** file in my lical directory with configuration. .evn file contain some const, links and Telegram Bot Token. 
I added **.env** file type to **.gitignore**, that is why it is not here.

If it is not what you asked for, lem me know.
Thanks

**Pull request merged**